src/renderer/containers/query-browser.jsx

```diff
@@ -154,6 +157,31 @@ class QueryBrowserContainer extends Component {
     dispatch(refreshDatabase(database));
   }
 
+  onShowDiagramModal(database) {
+    const { dispatch } = this.props;
+    dispatch(showDatabaseDiagram(database.name));
+  }
+
+  onShowDatabaseDiagram(database) {
+    const { dispatch } = this.props;
+    const selectedTables = [];
+
+    $(':checkbox:checked', 'div.ui.list').map((index, checkbox) => {
+      selectedTables.push(checkbox.id);
+    });
```

Comment:
This could be rewritten into this:
```js
const selectedTables = $(':checkbox:checked', 'div.ui.list')
  .map((index, checkbox) => checkbox.id);
```